lib/cmd/fh3/projects.js

```diff
@@ -112,6 +112,15 @@ function createProject(args, deployEnvironment, cb) {
     });
   }
 
+  // Like NGUI, filter out appTemplates which are
+  // not 'selected' before calling create
+  function filterSelectedApps(template) {
+    var filteredTemplate = _.clone(template);
```

Comment:
[Style hint] Verified that this template list is not used anywhere else and it's fetched every time we execute cmd. I think that we can just skip that clone operation.